src/Cns.ts

```diff
@@ -17,15 +17,16 @@ import {
 } from './publicTypes';
 import { isValidTwitterSignature } from './utils/TwitterSignatureValidator';
 import NamingService from './NamingService';
+import NetworkConfig from './config/network-config.json';
 
 export default class Cns extends EthereumNamingService {
   readonly contract: Contract;
   reader: ICnsReader;
   static TwitterVerificationAddress =
     '0x12cfb13522F13a78b650a8bCbFCf50b7CB899d82';
-  static ReaderMap: ReaderMap = {
-    1: '0x7ea9ee21077f84339eda9c80048ec6db678642b1',
-    42: '0xcf4318918fd18aca9bdc11445c01fbada4b448e3', // for internal testing
+  static ProxyReaderMap: ProxyReaderMap = { 
+    1: NetworkConfig.networks[1].contracts.ProxyReader.address, 
+    4: NetworkConfig.networks[4].contracts.ProxyReader.address, 
   };
```

Comment:
What if new networks will be added? you need to generate it with dynamic loop.

Reply:
Done ✅